documentation/deliverables/individual_deliverables/open-source-group-kickoff.md

```diff
@@ -0,0 +1,148 @@
+# Open Source Group Kickoff
+
+| Field           | Value                                                        |
+| --------------- | ------------------------------------------------------------ |
+| Document Status | Draft                                                        |
+| Epic Link       | [Issue 72](https://github.com/HHS/grants-equity/issues/72)   |
+| Epic Dashboard  | [Milestone Roadmap](https://github.com/orgs/HHS/projects/12) |
+
+
+## Short description
+
+-**What:** Set up the tools and processes needed to build an open source community around the Simpler Grants.gov initiative
+-**Why:** Ensures that open source contributors and the general public can easily participate in the project and provide input and code
+-**Who:** Any public members who want to contribute to the open source project. Focus is on: 
+- Open source contributors
+- Other external stakeholders
+
+## Goals
+
+### Business description & value
+
+**Problem statement:** Currently, the Simpler.Grants.gov project lacks a ready-to-use onboarding experience for individuals interested in joining the open source community. The absence of essential tools, processes, onboarding steps, and guidelines poses a challenge to meeting business goals and mission alignment. 
+
+**Value:** We seek to build stakeholder trust, promote project transparency, and establish a dynamic, participant-driven advisory council Launch an open source community with a primary emphasis on critical components that lay the groundwork for future expansion and productive collaboration, ensuring streamlined community management, fostering a vibrant open source community, and promoting project transparency.
+
+This effort allows us to... 
+
+- Set up the foundational set of communication channels for continuous user input, feedback, and engagement for an open source project
+- Ensure that public code contributions meet code quality and security standards
+- Build stakeholder trust in our product roadmap and our approach to development
+- Establish an ongoing Participant Advisory Council to guide future development
```

Comment:
Is this bullet more appropriate for the PAC Kickoff 30k?

Reply:
I can be more clear here. I was thinking if we wanted to have an open source group within the PAC, but maybe that's not needed or a priority for this.

Reply:
Agree that this seems a bit outside the scope of this 30k as it's currently defined, but a good goal to save for [that 30k](https://github.com/HHS/simpler-grants-gov/issues/327)